### ISSUE TITLE
[A11y] Work Show Page-All - Contrast

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1989]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

Example URLs:

- https://staging.oregondigital.org/concern/generics/df727084j
- https://staging.oregondigital.org/concern/documents/df727057n
- https://staging.oregondigital.org/concern/images/1g05fb620

Problems:

- All "success" badges (white text on green background) fail
  - "Public", toward the top of the page, just after the object name
- "REVIEW AND APPROVE" (white text on orange) fails
- Pagination "Prev" fails
- Pagination current page (white text on orange) fails

Additionally, the "success" div (when submitting a comment on a document) fails twice: the text itself and the "close" icon.

![image](https://user-images.githubusercontent.com/2157974/131517514-de7ce1a5-9344-427c-865b-ba2b641f6f3f.png)

I'm not sure of all the other possible alert types, but warnings look like they'll be problematic as well, and I suspect errors would also be.

### Suggested Fix

Fix color contrast in these areas...

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Run a tool like Accessibility insights and verify all of the above pass automated **and manual** contrast checks. Check against various alerts as well as the static page.

**Comment by azle** [comment]
Updated 12/3/2024

Above items were fixed, however new accessibility errors popped on Insights (6 errors) Wave (8 errors). I am not able to provide a pdf for some reason it won't print pdf correctly.

Insights
<img width="369" alt="Screenshot 2024-12-30 at 7 41 54 AM" src="https://github.com/user-attachments/assets/f7f1a365-0bd5-4c87-b24b-5375448297f9" />


WAVE Errors Preview
<img width="333" alt="Screenshot 2024-12-30 at 7 40 58 AM" src="https://github.com/user-attachments/assets/de8645cc-004e-4727-8ac0-788b178477fd" />
